examples/docs/go/.gitignore

```diff
@@ -0,0 +1,6 @@
+.shared
```

Comment:
This doesn't look like the gitignore for this directory, copied from somewhere else?